apps/minaintyg/src/pages/Certificate/components/CertificateActions/CertificateActions.tsx

```diff
@@ -21,26 +22,29 @@ export function CertificateActions({
   )
 
   return (
-    <div className="flex flex-col justify-end gap-5 md:flex-row">
-      <PrintCertificateAction id={id} availableFunctions={availableFunctions} />
-      {sendFunction && recipient && (
-        <>
-          <IDSButton
-            sblock
-            role="button"
-            onClick={() => {
-              if (recipient?.sent) {
-                updateShowCertificateSentDialog(true)
-              } else {
-                navigate('./skicka')
-              }
-            }}
-          >
-            {sendFunction.name}
-          </IDSButton>
-          <CertificateSentDialog recipient={recipient} open={showCertificateSentDialog} onOpenChange={updateShowCertificateSentDialog} />
-        </>
-      )}
-    </div>
+    <>
+      <CertificateInfoAlert availableFunctions={availableFunctions} />
```

Comment:
We could move this to `CertificatePage` and move tests to `CertificateInfoAlert.test.tsx`.